Update render.mdx (#8746)
withastro/docs: src/content/docs/en/guides/deploy/render.mdx

4. Give your website a name, select the branch and specify the build command and publish directory
   - **build command:** `npm run build`

>    - **publish directory:** `dist`, `dist/client` if you're using SSR. Make sure to include `export const prerender = true;` at the top of pages you want to serve as static.

Reviewer: ```suggestion
   - **publish directory:** `dist`, for static sites; `dist/client` if you're using `server` or `hybrid` server-rendering modes.
```
I just made these a little more explicitly stated! And, the advice about individual pages doesn't really seem go in this step where they are specifying things about the build. We don't need general advice about how to work with one of our on-demand rendering modes here in your set up instructions.
